webpack.mix.js

```diff
@@ -17,6 +17,6 @@ mix
   })
   .postCss('resources/css/entry.css', 'dist/css/', [postcss(), tailwindcss('tailwind.config.js')])
   .alias({
-    'laravel-nova': path.join(__dirname, '../../vendor/laravel/nova/resources/js/mixins/packages.js'),
```

Comment:
Note sure why this was using ../../

that didn't work for me

Reply:
Because as a nova component it is supposed to live in `newton-dashboard/nova-components/NovaNotesField`. And because `newton-dashboard/` already has nova installed - it's picking it from there.

Please rollback these '../../' changes